addons/vehicle_damage/XEH_postInit.sqf

```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+
+["ace_settingsInitialized", {
+    TRACE_1("settings init",GVAR(enabled));
+    if (GVAR(enabled)) then {
+        [QGVAR(bailOut), {
+            params ["_center", "_crewman", "_vehicle"];
+            TRACE_3("bailOut",_center,_crewman,_vehicle);
+
+            if (isPlayer _crewman) exitWith {};
+            if (!alive _crewman || { !( [_crewman] call EFUNC(common,isAwake))} ) exitWith {};
+
+            unassignVehicle _crewman;
+            _crewman leaveVehicle _vehicle;
+            doGetOut _crewman;
+
+            private _angle = floor (random 360);
+            private _dist = (30 + (random 10));
+            private _escape = _center getPos [_dist, _angle];
+
+            _crewman doMove _escape;
+            _crewman setSpeedMode "FULL";
+        }] call CBA_fnc_addEventHandler;
+
+        ["Tank", "init", LINKFUNC(addEventHandler), true, [], true] call CBA_fnc_addClassEventHandler;
+        ["Wheeled_APC_F", "init", LINKFUNC(addEventHandler), true, [], true] call CBA_fnc_addClassEventHandler;
+        
+        if (GVAR(enableCarDamage)) then {
+            ["Car", "init", LINKFUNC(addEventHandler), true, [], true] call CBA_fnc_addClassEventHandler;
+        };
+
+        // blow off turret effect
+        /*["Tank", "killed", {
+            if (random 1 < 0.15) then {
+                (_this select 0) call FUNC(blowOffTurret);
+            };
+        }, true, [], true] call CBA_fnc_addClassEventHandler;*/
```

Comment:
why is this commented out btw? not ready yet?

Reply:
Its buggy iirc

Reply:
The last time I tested, it was working. Also there is a param to disable it https://github.com/acemod/ACE3/blob/master/addons/cookoff/XEH_postInit.sqf#L87-L98

Reply:
I think I remember now: because this PR doesn't make vehicles destroyed, they can still be repaired. It doesn't make any sense to repair a tank that doesn't have a turret. A whole bunch of complications arise because of the assumption that the vehicle is dead when the turret blows off. What if there is still crew in the turret? What if the tank is repaired? And so on. I didn't want to solve these assumptions so I turned it off.

Reply:
Add that as comment in the code! :D